sr_ethercat_hand_config/launch/sr_rosbag_log.launch

```diff
@@ -7,10 +7,12 @@
 
   <arg name="log_max_splits" default="6"/>
   <arg name="log_directory" default="."/>
-  <arg name="log_bag_prefix" default="sr_hand_log"/>
+  <arg name="log_bag_prefix" default="sr_log"/>
 
   <arg name="log_location" value="$(arg log_directory)/$(arg log_bag_prefix)"/>
 
+  <arg name="clear_old_sessions" default="true"/>
+
   <arg name="log_topics" default='-e ".*(controller|debug_etherCAT_data).*" /joint_states /tf /tf_static /rosout /diagnostics /mechanism_statistics'/>
 
   <group unless="$(arg split_logs_by_size)">
```

Comment:
True, but if we're not splitting, we're consuming the whole hard disk, so.... ;) I get what you mean though. Unfortunately, I don't think there's any way to limit size without splitting, and I'm not sure if it's wise to have an option to just let it keep recording.